openpype/vendor/python/common/qargparse.py

```diff
@@ -570,7 +570,7 @@ def data(self, index, role):
         model = QtCore.QStringListModel(self["default"])
         widget = QtWidgets.QListView()
         widget.setModel(model)
-        widget.setEditTriggers(widget.NoEditTriggers)
+        widget.setEditTriggers(QtWidgets.QAbstractItemView.NoEditTriggers)
```

Comment:
Why is this needed? According to the documentation QListView inherits from `QtWidgets.QAbstractItemView`: https://doc.qt.io/qtforpython/PySide6/QtWidgets/QAbstractItemView.html

And thus should also have the attribute `NoEditTriggers`?

Reply:
I will quote what in description "PySide6 changed few things here and there. E.g. how it is accessed to class methods (objects don't have access to class attributes)..." If you don't trust me you can try install PySide6 and try it on your own :)